emails/utils.py

```diff
@@ -141,36 +123,48 @@ def ses_send_raw_email(
                 'Data': msg.as_string(),
             },
         )
-        incr_if_enabled('ses_send_email', 1)
+        incr_if_enabled('ses_send_raw_email', 1)
```

Comment:
Change the query on Grafana to pull from both `ses_send_raw_email` and `ses_send_email` otherwise we'll no longer have accurate count of emails relayed.

Reply:
We don't have any Grafana queries that use this yet, but I'll make sure we include both when we do.